.vscode/tasks.json

```diff
@@ -5,12 +5,13 @@
   "tasks": [
     {
       "type": "shell",
-      "label": "mocha: watch current file",
-      "command": "yarn mocha --watch ${fileDirname}/${fileBasename}",
+      "label": "mocha: test current file",
```

Comment:
this addresses an annoyance with the mocha watcher not watching for users who use the vscode build task. Currently the task is still 'running' but not picking up new changes, and trying to re-trigger it causes an annoying 'this task is still running' confirm box. I think the change is uncontroversial.